test/app_web/api_test.exs

```diff
@@ -124,7 +124,7 @@ defmodule AppWeb.APITest do
            }
   end
 
-  test "empty image (meaning it has a valid content type) should return an error", %{conn: conn} do
+  test "file with invalid binary data type and extension should return error. ", %{conn: conn} do
```

Comment:
This test should describe that, although the file is empty, it's still an image (`content-type` is `image/jpeg`, so the extension is not necessarily invalid.

Reply:
Thanks for fixing that test description. 👍

Reply:
I didn't fix anything, I'm mentioning the change you've made here might not be correct 👀